src/react-contenteditable.js

```diff
@@ -41,6 +42,7 @@ export default class ContentEditable extends React.Component {
       // Perhaps React (whose VDOM gets outdated because we often prevent
       // rerendering) did not update the DOM. So we update it manually now.
       this.htmlEl.innerHTML = this.props.html;
+      this.lastHtml = this.props.html;
```

Comment:
This assumes a controlled component. In the case where the developer only cares about the blur event, the html from props will not update with each keystroke and thus completely break the component's functionality.